### PR TITLE
Updated the installation chapter

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | #5627 |

### CODE REVIEW

File: book/installation.rst [hunk 1]
```diff
@@ -98,9 +98,6 @@ optional second argument of the ``new`` command:
     $ symfony new my_project_name 2.3.26
     $ symfony new my_project_name 2.6.5
 
-    # use the most recent LTS (Long Term Support) version
-    $ symfony new my_project_name lts
```
Comment (WouterJ) [inline book/installation.rst]: is this not supported?

Comment (javiereguiluz) [inline book/installation.rst]: Yes. In GitHub's split view it's easier to understand why I removed it (the explanation is duplicated!)

lts_repeated_explanation

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
:+1:

**Comment by WouterJ** [comment]
Actually, the 2.3 did not have a new release since this change was merged. So things will be a bit weird for people reading the 2.3 docs. But well, who is reading the 2.3 installation docs now that the 2.7 LTS is already released?

**Comment by xabbuh** [comment]
Which feature do you mean?

**Comment by WouterJ** [comment]
@xabbuh the new SE homepage

**Comment by javiereguiluz** [comment]
A new 2.3 version will be released "soon" and most newcomers start with 2.7 anyway, so I guess we can safely merge it as it is now.

**Comment by xabbuh** [comment]
Yeah, I think that's not really an issue since - like you said - people probably don't start new projects with 2.3 anymore.

**Pull request merged**